TYPO3.Media/Documentation/AsynchronousThumbnails.rst

```diff
@@ -20,7 +20,7 @@ see :ref:`Media ViewHelper Reference`.
 Configuration
 -------------
 
-The configuration setting ``TYPO3.Media.asyncThumbnails`` is used to determine if asynchronous thumbnails are rendered
+The configuration setting ``Neos.Media.asyncThumbnails`` is used to determine if asynchronous thumbnails are rendered
```

Comment:
The documentation will point to a non-existing setting (for now), so don't merge without actual vendor name change.